### ISSUE TITLE
Add function to hs.serial to get more detailed information about connected devices

### ISSUE THREAD

**Issue by latenitefilms** [Hammerspoon/hammerspoon#2862]
For example, using this NodeJS code:

```js
const SerialPort = require('serialport')
SerialPort.list().then(
  ports => ports.forEach(console.log),
  err => console.error(err)
)
```

It will return results such as:

```js
  {
    path: '/dev/tty.usbmodem132456781',
    manufacturer: 'Loupedeck',
    serialNumber: '13245678',
    pnpId: undefined,
    locationId: '14400000',
    vendorId: '2ec2',
    productId: '0003'
  }
 ```
 
 Having access to manufacturer, serial number, etc. would be very handy.

**Comment by latenitefilms** [comment]
See: https://github.com/armadsen/ORSSerialPort/wiki/Getting-Vendor-ID-and-Product-ID

**Comment by latenitefilms** [comment]
Closing - will be solved in #2871
